code/controllers/master_controller.dm

```diff
@@ -23,19 +23,20 @@ var/global/datum/controller/game_controller/master_controller = new()
 			init_subtypes(/datum/subsystem, subsystems)
 
 		master_controller = src
+	calculateGCD()
 
+/datum/controller/game_controller/proc/calculateGCD()
 /*
 calculate the longest number of ticks the MC can wait between each cycle without causing subsystems to not fire on schedule
-Note: you can set the datum's defined processing_interval to some integer to set an -absolute- minimum wait duration.
 */
 	var/GCD
 	for(var/datum/subsystem/SS in subsystems)
 		if(SS.wait)
 			GCD = Gcd(SS.wait, GCD)
 	GCD = round(GCD)
-	if(GCD > processing_interval)
-		processing_interval = GCD
-
+	if(GCD < 1)
+		GCD = 1
+	processing_interval = GCD

```

Comment:
This whole block is entirely unnecessary. And frankly it shouldn't be 1. The lowest value should be world.tick_lag. The reason for this is simple: if we go to 20 fps or higher, it would be nice to take advantage of every other tick, which would currently be wasted.